package.json

```diff
@@ -25,12 +25,12 @@
     "test:ember": "ember test"
   },
   "overrides": {
-    "@appuniversum/ember-appuniversum": {
+    "@lblod/ember-environment-banner": {
```

Comment:
v0.5.0 doesn't support ember-source v5 yet. Once https://github.com/lblod/ember-environment-banner/pull/73 is merged and released we can remove this override again.